lightautoml/addons/interpretation/l2x_model.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
set the exact rule to ignore

Reply:
Specified error codes in the configuration file.

Reply:
Please check "flake8: noqa" in all files. I think, I haven't highlighted all of them.

Reply:
fix